### ISSUE TITLE
Missing conv bias

### ISSUE THREAD

**Issue by amandayeyan** [KaiyangZhou/deep-person-reid#162]
Hi @KaiyangZhou ,
When loading the pretrained model, why some conv layer has bias and some conv layer does not? Is that what you purposely designed in this way?

**Comment by KaiyangZhou** [comment]
any specific example?

**Comment by amandayeyan** [comment]
> any specific example?

In your mobilenet network, class ConvBlock(nn.Module) defined bias=False, and in class Bottleneck(nn.Module), all Conv2d defined bias=False, that would not save any bias for convolution layer.
However, when print out list(model.items()), there is bias values saved for each conv3 in Bottleneck, but all neat to zero as I checked. 

FYI~

**Comment by amandayeyan** [comment]
Correction. Eventually, it is realized all bias bias of conv layers are off. The bias terms are all for batchnorm layers.
